src/Hashids.php

```diff
@@ -91,6 +102,7 @@ public function __construct($salt = '', $minHashLength = 0, $alphabet = 'abcdefg
         $this->salt = $salt;
         $this->minHashLength = $minHashLength;
         $this->alphabet = implode('', array_unique(str_split($alphabet)));
+        $this->math = MathFactory::create();
```

Comment:
Does the `::create()` method have to be static? Maybe we should convert it to just create a new `MathFactory` instance?

Reply:
The only reason `MathFactory` existed was so I could re-use the logic in `Hashids\Math`, but now that it's being removed I can just move the `::create` logic directly into `Hashids\Hashids`.